lib/httpClient.js

```diff
@@ -20,6 +20,17 @@ const axios = require('axios')
 const httpsClient = require('https')
 const Pino = require('pino')
 
+const defaultRedactionRules = {
```

Comment:
these values are duplicated between lc39 and here; can't we import from there? 🤔 

https://github.com/mia-platform/lc39/pull/238/files#diff-d57c729610512231f69d0ec254cb098adbc62d05db62e16961b4d5920b2c69e6R22

Reply:
@davidebianchi what do you think about it?